maya-io-server-staging-1589895026/templates/dep-alertstore-tablemanager.yaml

```diff
@@ -45,7 +48,7 @@ spec:
           done
       containers:
       - name: alertstore-tablemanager
-        image: mayadataio/alertstore-tablemanager:staging-v1.11.15
+        image: mayadataio/alertstore-tablemanager:{{ .Values.server.release }}
```

Comment:
Replace the above image with `mayadataio/alertstore-tablemanager:staging-v1.11.15`